### DISCUSSION TITLE
The accuracy between “learn.recorder.plot_metrics()” and munually computed from "plot_confusion_matrix()" are NOT THE SAME

### DISCUSSION THREAD

**Discussion by jazzylee** [timeseriesAI/tsai#113] [Q&A]
Recently I‘ve begun to use **tsai**,but I got confused by the accuracy.
When I use the "learn.recorder.plot_metrics()" method,a figure containing accuracy is shown,which peforms very well.
But when I draw the confusion matrix using "plot_confusion_matrix()" and compute the accuracy manually,the result is not the same as the above way.
So how can I get the accuracy and why the two ways mentioned above are different?

**Comment by vrodriguezf** [comment]
Hi @jazzylee , can you show an example?

**Reply by jazzylee** [reply]
Hi,vrodriguezf,thanks for your replying!😁
I tried to apply InceptionTime in my project and found the issue.Here's some core screenshots below.
accuracy from acc figure:
![image](https://user-images.githubusercontent.com/33250251/117832349-fdaa2180-b2a7-11eb-8b6b-1739311b2b64.png)
accuracy calculated from confusion matrix:
![image](https://user-images.githubusercontent.com/33250251/117833370-df90f100-b2a8-11eb-8f41-f2dbb6d1ea87.png)

**Reply by oguiza** [reply]
Hi @jazzylee, 
I'm sorry about the late reply. 
I've tried to replicate this issue, and have tried different settings, but I couldn't. I always get the same result. This is the code that I've used: 
```
from tsai.all import *

dsid = 'ECG200'
X, y, splits = get_UCR_data(dsid, split_data=False)
tfms = [None, TSClassification()]
dls = get_ts_dls(X, y, splits=splits, tfms=tfms)
learn = ts_learner(dls, TST, metrics=accuracy, cbs=ShowGraph())
learn.fit_one_cycle(10, 1e-4)
interp = ClassificationInterpretation.from_learner(learn)
interp.plot_confusion_matrix()
print(f'from train: {learn.recorder.values[-1][-1]:.3f} from interp: {np.trace(interp.confusion_matrix()) / interp.confusion_matrix().sum():.3f}')


# output
# from train: 0.730 from interp: 0.730
```
Do you still get these conflicting results?
